spec/lib/xclisten_spec.rb

```diff
@@ -86,7 +88,7 @@ def run(args)
         :workspace => 'Example/Sample.xcworkspace',
         :scheme => 'kif',
         :sdk => 'iphoneos',
-        :device => 'name=iPhone Retina (4-inch 64-bit)'
+        :device => 'name=iPhone 5s'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.